package it.nextdevs.EsercizioS2G3.exception;

public class CentralizedExceptionError {
}
